_data/topnav.yml

```diff
@@ -1,13 +1,10 @@
 ## 
 ## if you want to list an external url, use external_url instead of url. the theme will apply a different link base.
 subitems:
-  - title: Main
-    url: /index
-  - title: Lifecycle
```

Comment:
It was decided that the navbar button `Lifecycle` should instead be `Training lifecycle`.